pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -173,18 +176,30 @@ func askUsername() error {
 }
 
 // askCredentials will handle username and password for connecting with Engine
-// In case of error during password, users will be prompted username again.
+// max attempts is defined by maxAttemptsLogin var
 func askCredentials() error {
-	err := askUsername()
-	if err != nil {
-		return err
-	}
 
-	err = askPassword()
-	if err != nil {
-		return askUsername()
-	}
+	// Record to number of attempts to login
+	var loginAttempts = 0
 
+	for {
```

Comment:
How about something like that:

	loginAttempts := 3
	for loginAttempts > 0 {
		err := askUsername()
		if err != nil {
			return err
		}

		err = askPassword()
		if err != nil {
			return nil
		}
		loginAttempts = loginAttempts-1
	}

?

Reply:
looks good to me, let me prepare an update.